stretch_core/stretch_core/stretch_driver.py

```diff
@@ -591,41 +599,50 @@ def main(self):
         elif mode == "manipulation":
             self.turn_on_manipulation_mode()
 
-        self.switch_to_manipulation_mode_service = rospy.Service('/switch_to_manipulation_mode',
-                                                                 Trigger,
-                                                                 self.manipulation_mode_service_callback)
+        self.switch_to_manipulation_mode_service = self.create_service(Trigger,
+                                                                       '/switch_to_manipulation_mode',
+                                                                       self.manipulation_mode_service_callback)
+
+        self.switch_to_navigation_mode_service = self.create_service('/switch_to_navigation_mode',
```

Comment:
`create_service` takes the service type as the 1st parameter and name as the 2nd parameter, same for the next 3 services